docs/user_guide/ttl.md

```diff
@@ -14,10 +14,34 @@ store-version, after the push finishes, but before swapping the read traffic ove
 replay phase where the last N seconds of buffered real-time data is replayed. The replayed data gets overlaid on top of
 the data from the Full Push.
 
-This can be leveraged in order to achieve a TTL behavior. By scheduling periodic empty pushes and configuring how far 
-back to replay the real-time data (N), one can control the TTL parameters. The time to live is defined by N, which acts 
-as a "minimum TTL", while the "maximum TTL" is N + delay between each empty push. For example, if you schedule a daily 
-empty push, and N = 6 days, then the oldest data in your store will be at least 6 days old, and at most 7 days old.
+This can be leveraged in order to achieve a TTL behavior. Currently, there are two options that leverages this mechanism. 
+
+### Empty push
+By scheduling periodic empty pushes and configuring how far back to replay the real-time data (N), one can control the 
+TTL parameters. The time to live is defined by N, which acts as a "minimum TTL", while the "maximum TTL" is N + delay 
+between each empty push. For example, if you schedule a daily empty push, and N = 6 days, then the oldest data in your 
+store will be at least 6 days old, and at most 7 days old.
+
+The N is the `hybrid-rewind-seconds`, that's a part of hybrid store configuration.
+
+### [Experimental] Repush with TTL
+Similarly to empty push, the repush with TTL can also be configured and scheduled periodically to achieve the TTL behavior.
+The major differences between this and empty push are original data source and how real-time buffer is replayed.
+
+|                 | Data origin            | Real-time buffer replay                                               |
+|-----------------|------------------------|-----------------------------------------------------------------------|
+| Empty push      | None                   | Replay real-time buffer with `hybrid-rewind-seconds` config           |
+| Repush with TTL | Existing version topic | Replay real-time buffer with `rewind.time.in.seconds.override` config |
+
+The `rewind.time.in.seconds.override` is a configurable value in push job, default to 24 hours.
+
+
+This brings two major benefits:
+1. The repush job de-dupes writes to the same key, so that the servers need to ingest fewer events.
+2. The repush job sorts the data, thus allowing servers to ingest in a more optimized way.
+
+Please note that this feature only supports Venice stores that enable active-active replication and doesn't enable 
+write compute yet, empty push doesn't have this requirement.
```

Comment:
may be dont need to specify WC part, as write compute AA will be GA soon, in that case we would have to update this again.

Reply:
I think it's fine to re-update the doc as often as necessary... we should document the current state. Future work, if mentioned at all, should be marked explicitly as future.